pkg/apis/flowcontrol/install/install.go

```diff
@@ -38,6 +39,11 @@ func Install(scheme *runtime.Scheme) {
 	utilruntime.Must(flowcontrolv1beta1.AddToScheme(scheme))
 	utilruntime.Must(flowcontrolv1beta2.AddToScheme(scheme))
 	utilruntime.Must(flowcontrolv1beta3.AddToScheme(scheme))
-	utilruntime.Must(scheme.SetVersionPriority(flowcontrolv1beta3.SchemeGroupVersion, flowcontrolv1beta2.SchemeGroupVersion,
-		flowcontrolv1beta1.SchemeGroupVersion))
+	utilruntime.Must(flowcontrolv1.AddToScheme(scheme))
+
+	// TODO(#121119): This controls serialization order, for 1.29, we continue
+	//  to use v1beta3 as the serialization version because vN-1 understands that
+	//  level. In 1.30, we should set the serialization version to v1.
+	utilruntime.Must(scheme.SetVersionPriority(flowcontrolv1beta3.SchemeGroupVersion, flowcontrolv1.SchemeGroupVersion,
+		flowcontrolv1beta2.SchemeGroupVersion, flowcontrolv1beta1.SchemeGroupVersion))
```

Comment:
question: can we remove `v1beta1` and `v1beta2` from the version priority list,
since `v1beta1` has been removed in 1.26, and `v1beta2` will be removed in 1.29?

Reply:
agreed, leave this to support old clusters